docs/upgrade-notes.asciidoc

```diff
@@ -0,0 +1,718 @@
+
+*Impact* +
+This impacts user roles that have *Privileges for all features* set to *All*. After this update, users with this role will have access to the Osquery page in {kib}. However, to use the Osquery feature fully, these requirements remain the same: users also need Read access to the logs-osquery_manager.result* index and the Osquery Manager integration must be deployed to Elastic Agents.
+====
+
+[discrete]
+.Removed legacy CSV export type. (8.1)
+[%collapsible]
+====
+*Details* +
+The `/api/reporting/generate/csv` endpoint has been removed. For more information, refer to {kibana-pull}121435[#121435].
+
+*Impact* +
+If you are using 7.13.0 and earlier, {kibana-ref-all}/8.1/automating-report-generation.html[regenerate the POST URLs] that you use to automatically generate CSV reports.
+====
+
+[discrete]
+.Removed legacy PDF shim. (8.1)
+[%collapsible]
+====
+*Details* +
+The POST URLs that you generated in {kib} 6.2.0 no longer work. For more information, refer to {kibana-pull}121369[#121369].
+
+*Impact* +
+{kibana-ref-all}/8.1/automating-report-generation.html[Regenerate the POST URLs] that you use to automatatically generate PDF reports.
+====
+
+[discrete]
+.Removed the `console.ssl` setting. (8.0)
+[%collapsible]
+====
+*Details* +
+The `console.ssl` setting has been removed. For more information, refer to {kibana-pull}123754[#123754].
+
+*Impact* +
+Before you upgrade to 8.0.0, remove `console.ssl` from kibana.yml.
+====
+
+[discrete]
+.Removed support for `monitoring.cluster_alerts.allowedSpaces`. (8.0)
+[%collapsible]
+====
+*Details* +
+The `monitoring.cluster_alerts.allowedSpaces` setting, which {kib} uses to create Stack Monitoring alerts, has been removed. For more information, refer to {kibana-pull}123229[#123229].
+
+*Impact* +
+Before you upgrade to 8.0.0, remove `monitoring.cluster_alerts.allowedSpaces` from kibana.yml.
+====
+
+[float]
+=== Deprecation notices
+
+The following functionality is deprecated and will be removed at a future date. Deprecated functionality 
+does not have an immediate impact on your application, but we strongly recommend you make the necessary 
+updates to avoid use of deprecated features.
+
+Use the **Kibana Upgrade Assistant** to prepare for your upgrade to the next version of the Elastic Stack. 
+The assistant identifies deprecated settings in your configuration and guides you through the process of 
+resolving issues if any deprecated features are enabled. 
+To access the assistant, go to **Stack Management** > **Upgrade Assistant**.
+
+[discrete]
+.Uptime is deprecated in 8.15.0 and will be removed in a future version. (8.15)
+[%collapsible]
+====
+*Details* +
+The Uptime app is already hidden from Kibana when there is no recent Heartbeat data, and will be completely removed in early 2026. You should migrate to Synthetics as an alternative. For more details, refer to the {observability-guide}/uptime-intro.html[Uptime documentation].
+====
```

Comment:
```suggestion
[discrete]
.Uptime is deprecated in 8.15.0. (8.15)
[%collapsible]
====
*Details* +
The Uptime app is already hidden from Kibana when there is no recent Heartbeat data. Migrate to Synthetics as an alternative. For more details, refer to the {observability-guide}/uptime-intro.html[Uptime documentation].
====
```
Removing information about Uptime removal as per Sophie's note in the linked issue.